code/modules/medical/autopsy.dm

```diff
@@ -156,18 +136,23 @@
 		scan_data += "Possible weapons:<br>"
 		for(var/weapon_name in weapon_chances)
 			scan_data += "\t[100*weapon_chances[weapon_name]/total_score]% [weapon_name]<br>"
-
 		scan_data += "<br>"
-
 		n++
-
 	if(chemtraces.len)
 		scan_data += "<b>Trace Chemicals: </b><br>"
 		for(var/chemID in chemtraces)
 			scan_data += chemID
 			scan_data += "<br>"
-
 	scan_data += "<br>[advanced_butchery]<br>"
+	return scan_data
+
+/obj/item/weapon/autopsy_scanner/verb/print_data()
+	set category = "Object"
+	set src in view(usr, 1)
+	set name = "Print Data"
+	if(usr.isUnconscious() || !(istype(usr,/mob/living/carbon/human)))
+		to_chat(usr, "No.")
```

Comment:
this doesn't even have a span and bad feedback

Reply:
same here